server/channels/api4/report.go

```diff
@@ -28,6 +28,11 @@ func getUsersForReporting(c *Context, w http.ResponseWriter, r *http.Request) {
 		sortColumn = r.URL.Query().Get("sort_column")
 	}
 
+	direction := "down"
```

Comment:
We should probably move these to constants in the `web/params.go` file. Something for later.